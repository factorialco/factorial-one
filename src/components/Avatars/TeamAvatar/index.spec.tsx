import { render } from "@testing-library/react-native";
import React from "react";
import { TeamAvatar } from "./";

describe("TeamAvatar", () => {
  it("Snapshot", () => {
    const { toJSON } = render(<TeamAvatar name="Factorial One" />);

    expect(toJSON()).toMatchSnapshot();
  });
});
