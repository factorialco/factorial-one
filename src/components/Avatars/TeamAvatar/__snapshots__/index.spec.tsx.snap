// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamAvatar Snapshot 1`] = `
<View
  className="flex inline-flex "
>
  <View
    className="h-fit w-fit"
  >
    <View
      aria-hidden={true}
      className="flex shrink-0 items-center justify-center overflow-hidden text-center font-semibold w-8 h-8 rounded bg-[hsl(theme(colors.smoke.50))]"
      data-a11y-color-contrast-ignore={true}
      role="img"
    >
      <Text
        className="text-f1-foreground-inverse/90 text-md"
        data-a11y-color-contrast-ignore={true}
      >
        FA
      </Text>
    </View>
  </View>
</View>
`;
